SU-SWS/stanford-jumpstart-deployer
# Add node_revision_delete-7.x-2.7

# READY FOR REVIEW


# Summary
- Adds [Node Revision Delete](https://drupal.org/project/node_revision_delete)

# Needed By (Date)
- The horse has already left the barn (it's already been added to the Sites stack)

# Criticality
- How critical is this PR on a 1-10 scale? 1

# Steps to Test

1. Build 'em all
2. Ensure they build
3. Go, Travis, go!

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? It helps with trimming down database size due to runaway Course revisions.

# Associated Issues and/or People
- https://github.com/SU-SWS/Stanford-Drupal-Profile/pull/100
- Anyone who should be notified? ( @pookmish )

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
